hca/submission.py

```diff
@@ -3,15 +3,36 @@
 from json_converter.json_mapper import JsonMapper
 from submission_broker.submission.submission import Submission, Entity, HandleCollision
 
+from hca.accession_mapper import AccessionMapper
```

Comment:
some children with communication difficulties have hugely benefitted from using specialised apps and they use them as their main way of communication.

Reply:
I am not sure I understand what you mean. `HcaSubmission` class not only dealing with accessions. It is dealing with a submission and its related entities.

Reply:
I think I did this as I moved some part of the code to the `AccessionMapper` class.